manifests/image-references

```diff
@@ -0,0 +1,28 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: cluster-logging-operator
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/cluster-logging-operator:latest
+  - name: elasticsearch-operator
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-elasticsearch-operator:latest
+  - name: kibana
```

Comment:
lets add the version to this name, and prefix with `logging-` perhaps for image stream groupings